src/pages/Bridge/index.tsx

```diff
@@ -27,46 +43,108 @@ import {
   useSwitchChain,
 } from 'packages/wallet';
 import { ChainId } from 'types';
+import {
+  convertDollarsToCents,
+  convertMantissaToTokens,
+  convertTokensToMantissa,
+  formatCentsToReadableValue,
+  formatTokensToReadableValue,
+} from 'utilities';
 
 import { ChainSelect } from './ChainSelect';
 import { ReactComponent as LayerZeroLogo } from './layerZeroLogo.svg';
 import TEST_IDS from './testIds';
 
+const BRIDGE_DOC_URL =
+  'https://docs-v4.venus.io/technical-reference/reference-technical-articles/technical-doc-xvs-bridge';
+
 const formSchema = z.object({
-  fromChainId: z.number(),
-  toChainId: z.number(),
-  amountTokens: z.string().min(1),
+  doesNotHaveEnoughNativeFunds: z.boolean().refine((v: boolean) => !v),
+  isDailyTransactionLimitExceeded: z.boolean().refine((v: boolean) => !v),
+  isSingleTransactionLimitExceeded: z.boolean().refine((v: boolean) => !v),
```

Comment:
The form schema should not contain fields for errors. Instead, you should handle errors and their messages through the validation rules of each field (in this case with the validation rules of `amountTokens` field): https://zod.dev/?id=strings

So for each of these errors you'd define a rule that you attach to `amountTokens` with a corresponding error message when it is not being respected.